rust/foxglove/src/websocket.rs

```diff
@@ -774,6 +810,18 @@ impl ConnectedClient {
         self.send_control_msg(msg);
     }
 
+    fn on_fetch_asset(&self, server: Arc<Server>, uri: String, request_id: u32) {
```

Comment:
Services have a `call_id`; should this be consistent? Or is this semantically different? (I like request_id better.)

Reply:
I'm copying it from the spec where it's called requestId: https://github.com/foxglove/ws-protocol/blob/main/docs/spec.md#fetch-asset

Maybe they should be named the same.

Reply:
👍 I like request_id; I do think we should consider renaming `call_id` for consistency, but will see what @gasmith thinks.

Reply:
I like Greg's suggestion